src/xcode/ENA/ENA/Source/Scenes/ErrorReporting/ErrorReportCoordinator.swift

```diff
@@ -91,26 +98,30 @@ final class ErrorReportsCoordinator: ErrorReportsCoordinating {
 				}
 			)
 		)
-		
 		let topBottomViewController = TopBottomContainerViewController(
 			topController: topViewController,
 			bottomController: bottomViewController,
 			bottomHeight: 80
 		)
+		
 		rootViewController.navigationController?.pushViewController(topBottomViewController, animated: true)
-
 	}
 	
 	private let rootViewController: UIViewController
 	private var errorLoggingStatus: ErrorLoggingStatus
-
+	private var hasHistory: Bool = false
```

Comment:
Why do we need this?

Reply:
We don't, i thought we will need it to add the history cell or not, i will remove it